test/detailed-results-MD041-MD050.md

```diff
@@ -24,4 +24,8 @@ Fenced code
 
     Indented code
 
-Missing newline character
\ No newline at end of file
+Missing newline character
+
+~~~text
+Fenced code
+~~~
```

Comment:
Please move this above the "Missing newline character" line so that remains accurate and doesn't risk two violations on the last line.